content/first-order-logic/models-theories/theories.tex

```diff
@@ -126,8 +126,8 @@
         \Part{y}{z}) \lif \Part{x}{z})]]],\\
 \intertext{Moreover, any two objects have a fusion (an object that has
   only these two objects and all their parts as parts).}  &
-\lforall[x][\lforall[y][\lexists[z][\lforall[u][(\Part{u}{z} \liff
-        (\Part{u}{x} \land \Part{u}{y}))]]]]
+\lforall[x][\lforall[y][\lexists[z][\lforall[u][(\Part{z}{u} \liff
```

Comment:
Something (u) is a part of the fusion of x and y iff it's a part of x or of y. So there should be an \lor instead of a \land, but the order is right. No?